SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialDebugger.h

```diff
@@ -134,6 +137,11 @@ class SPATIALGDK_API ASpatialDebugger : public AInfo
 	void ActorAuthorityChanged(const Worker_AuthorityChangeOp& AuthOp) const;
 	void ActorAuthorityIntentChanged(Worker_EntityId EntityId, VirtualWorkerId NewIntentVirtualWorkerId) const;
 
+	void EditorInitialise();
```

Comment:
Seems like we could wrap all this functionality in WITH_EDITOR. Any reason not to?

Reply:
Yes, I think that is a good idea. I have wrapped up all these functions up in WITH_EDITOR.